lib/utils/group-helper.js

```diff
@@ -57,8 +57,7 @@ class GroupHelper {
     }
     mapArgToBoolean(name, optionName) {
         ifArg(name, function(bool) {
-            if (bool === true) options[optionName || name] = true;
-            else if (bool === false) options[optionName || name] = false;
+            options[optionName || name] = bool;
```

Comment:
What if `bool` is not a real boolean?

Reply:
The check assumes `bool` to be either `true` or `false` right?

Reply:
Oh yeah, making sure none other than `true` or `false` comes up right?

Reply:
Yes exactly. Your change changes the functionality of the code.